packages/notifications/push/amplify_push_notifications_pinpoint/lib/src/push_notifications_background_processing.dart

```diff
@@ -40,13 +45,14 @@ Future<void> amplifyBackgroundProcessing({
   }
   final notificationsPlugin = AmplifyPushNotificationsPinpoint();
   final authPlugin = AmplifyAuthCognito();
-  if (!Amplify.isConfigured) {
-    await Amplify.addPlugins([authPlugin, notificationsPlugin]);
-    await Amplify.configure(amplifyconfigStr);
+  final mockableAmplify = amplify ?? Amplify;
```

Comment:
I'm unsure what happened to my previous comments about including mockable prefix in the code.  
It's strange to see testing code mixed with production code here.  I haven't seen this anywhere else in our codebase and it confuses from the main purpose of your code. 

For example, is the below just for when you mock or for production cases as well?  What information are you conveying with the word 'mockable' that isn't already clear by adding @visibleForTesting above?

Reply:
It's only for testing purposes. I agree it's a bad idea to mix these. I'll update them.